osidb/tests/test_endpoints.py

```diff
@@ -1145,6 +1146,41 @@ def test_flaw_update(self, auth_client, test_api_uri):
         assert "appended test title" in body["title"]
         assert original_body["description"] == body["description"]
 
+    def test_flaw_update_collision(self, auth_client, test_api_uri):
+        """
+        test that updating a flaw while sending an outdated updated_dt
+        timestamp is correctly recognized as mid-air collision and rejected
+        """
+        flaw = FlawFactory(embargoed=False)
+        AffectFactory(flaw=flaw)
+
+        response = auth_client.put(
+            f"{test_api_uri}/flaws/{flaw.uuid}",
+            {
+                "uuid": flaw.uuid,
+                "cve_id": flaw.cve_id,
+                "type": flaw.type,
+                "title": f"{flaw.title} appended test title",
+                "description": flaw.description,
+                "state": flaw.state,
+                "resolution": flaw.resolution,
+                "impact": flaw.impact,
+                "source": flaw.source,
+                "embargoed": flaw.embargoed,
+                "updated_dt": flaw.updated_dt - timedelta(days=1),  # outdated timestamp
+                "bz_api_key": "SECRET",
```

Comment:
use header instead of body request

Reply:
Again the old code here...the conflicts below would force me anyway :smile: